### ISSUE TITLE
fsoinstaller is unable to download mod.ini

### ISSUE THREAD

**Issue by Spongman** [scp-fs2open/fsoinstaller#12]
there's a bug in the Apache installation on `www.fsoinstaller.com` that's preventing it from serving valid requests from the `FreeSpaceOpenInstaller.jar` program.

The program output is
```
2023-02-25 01:05:37,555 [pool-1-thread-4] ERROR com.fsoinstaller.internet.Downloader - An exception was thrown during download!
java.io.IOException: Server returned HTTP response code 500 for URL http://www.fsoinstaller.com/files/installer/java/mod.ini
        at com.fsoinstaller.internet.Connector.getContentLengthImpl(Connector.java:268)
        at com.fsoinstaller.internet.Connector.getContentLength(Connector.java:294)
        at com.fsoinstaller.internet.Downloader.downloadFile(Downloader.java:308)
        at com.fsoinstaller.internet.Downloader.download0(Downloader.java:252)
        at com.fsoinstaller.internet.Downloader.download(Downloader.java:159)
        at com.fsoinstaller.wizard.InstallItem.downloadOne(InstallItem.java:1318)
        at com.fsoinstaller.wizard.InstallItem.access$1900(InstallItem.java:76)
        at com.fsoinstaller.wizard.InstallItem$7.call(InstallItem.java:931)
        at com.fsoinstaller.wizard.InstallItem$7.call(InstallItem.java:924)
        at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
        at java.base/java.lang.Thread.run(Thread.java:833)
```

wireshark shows the request:
```
GET /files/installer/java/mod.ini HTTP/1.1
User-Agent: Mozilla/5.0 (X11; Linux x86_64; rv:34.0) Gecko/20100101 Firefox/28.0
Host: www.fsoinstaller.com
Accept: text/html, image/gif, image/jpeg, *; q=.2, */*; q=.2
Connection: keep-alive

HTTP/1.1 500 Internal Server Error
Date: Sat, 25 Feb 2023 09:05:38 GMT
Server: Apache
Content-Length: 539
Keep-Alive: timeout=5, max=100
Connection: Keep-Alive
Content-Type: text/html; charset=iso-8859-1

<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head>
<title>500 Internal Server Error</title>
</head><body>
<h1>Internal Server Error</h1>
<p>The server encountered an internal error or
misconfiguration and was unable to complete
your request.</p>
<p>Please contact the server administrator at 
 webmaster@fsoinstaller.com to inform them of the time this error occurred,
 and the actions you performed just before this error.</p>
<p>More information about this error may be available
in the server error log.</p>
</body></html>
```

that's a completely valid http request, but the web server is barfing on it.

**Comment by Spongman** [comment]
it looks like it's the following http heaader that's causing the issue:
```
Accept: text/html, image/gif, image/jpeg, *; q=.2, */*; q=.2
```

**Comment by Spongman** [comment]
repro:
```
c:\>curl.exe -v -H "Accept: text/html, image/gif, image/jpeg, *; q=.2, */*; q=.2" http://www.fsoinstaller.com/files/installer/java/mod.ini
*   Trying 208.113.213.97:80...
* Connected to www.fsoinstaller.com (208.113.213.97) port 80 (#0)
> GET /files/installer/java/mod.ini HTTP/1.1
> Host: www.fsoinstaller.com
> User-Agent: curl/7.83.1
> Accept: text/html, image/gif, image/jpeg, *; q=.2, */*; q=.2
>
* Mark bundle as not supporting multiuse
< HTTP/1.1 500 Internal Server Error
< Date: Sat, 25 Feb 2023 09:33:54 GMT
< Server: Apache
< Content-Length: 539
< Content-Type: text/html; charset=iso-8859-1
<
<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
```

**Comment by Spongman** [comment]
apparently `"Accept: *; q=.2"` is sufficient to break Apache. Apparently it's a bug in java: https://bugs.openjdk.org/browse/JDK-8163921

**Comment by Goober5000** [comment]
Excellent detective work!  I had noticed this myself but was not sure how to troubleshoot it or where to start looking.

**Comment by Goober5000** [comment]
Fixed by #13
